app/models/user.rb

```diff
@@ -20,10 +21,12 @@
 #  index_users_on_confirmation_token    (confirmation_token) UNIQUE
 #  index_users_on_email                 (email) UNIQUE
 #  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_username              (username) UNIQUE
 #
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable, :confirmable
+  validates :username, presence: true, format: { with: /\A[A-Za-z0-9]+\z/}
```

Comment:
What is the purpose of having a username?

Asking bc I think email it's enough, and if we plan to use it for sign-in, then we need this to be unique as well.

It'd be nice if we can get rid of it and rely on the email. WDYT? 😄

Reply:
We added it because it was specified in the requirements of the project. At the beginning I didn't understand why make it alphanumeric and unique if wasn't going to be used for signing in with it. But then, I realized we could use it to be displayed in the complaints instead of using the real name or email.

Reply:
Although, we can also allow anonymous complaints, perhaps we can remove it if we find it useless, WDYT?